lib/ansible/plugins/action/service.py

```diff
@@ -68,6 +68,7 @@ def run(self, tmp=None, task_vars=None):
 
             # for backwards compatibility
             if 'state' in new_module_args and new_module_args['state'] == 'running':
+                self._display.deprecated(msg="state=running is deprecated. Please use state=started", version="2.4")
```

Comment:
not a big issue for sure. I thought since we now have all the tools to deprecate features which are there "for backwards compatibility". Deprecating it wouldn't hurt.